src/axom/slam/policies/StridePolicies.hpp

```diff
@@ -51,12 +51,19 @@ struct RuntimeStride
 public:
   static const IntType DEFAULT_VALUE = IntType(1);
   static const bool IS_COMPILE_TIME = false;
+  constexpr static int NumDims = 1;
+
+  using IndexType = IntType;
+  using ShapeType = IntType;
+
+  static constexpr IntType DefaultSize() { return DEFAULT_VALUE; }
 
   AXOM_HOST_DEVICE RuntimeStride(IntType stride = DEFAULT_VALUE)
     : m_stride(stride)
   { }
 
   AXOM_HOST_DEVICE inline IntType stride() const { return m_stride; }
+  AXOM_HOST_DEVICE inline IntType shape() const { return m_stride; }
```

Comment:
Is this a typo?  Should it return `m_shape`?

Reply:
Not a typo. Perhaps a comment would help.

Reply:
Added some comments to clarify.